### PR TITLE
Privacy wrapper for video & applets

### LINKED ISSUE #631: Add privacy wrapper for video and applets
### TODO 🏁 
- [ ] Implement CF Worker Magic 🧙([tracked here](https://github.com/serlo/serlo.org-cloudflare-worker/issues/66))
- [ ] Add real wording for notice (@inyono ?)
- [ ] Internationalize component (@elbotho)

### How it should work

<details>
<summary>Steps</summary>

- Lazy load the whole thing
- Then get some preview image proxied by the CF Worker
- Display preview Image with play button or other icon for applets
- On Click/Tap/Enter: Load iframe from external service
- Even nicer: get rid of second click to play if possible. (not implemented for wikimedia)
- Use local storage to save decision for this provider?
</details>

### Backlog / TODO later
- Idea: For SEO we could use our lazyload SEO solution and load the iframe directly in the noscript section.
- Geogebra: Get preview image somehow. 
- BR currently not supported and used. might be added later

### Notes
<details>
  <summary>Thumbnail links / autoplay ideas</summary>
  
  **Wikimedia**
[Video link](https://upload.wikimedia.org/wikipedia/commons/1/15/Inerter_vibration_isolation_experiment.webm) -> [Thumbnail link](https://upload.wikimedia.org/wikipedia/commons/thumb/1/15/Inerter_vibration_isolation_experiment.webm/1000px--Inerter_vibration_isolation_experiment.webm.jpg)
structure should be: 
`'upload.wikimedia.org/wikipedia/commons' + '/thumb' + '/1/15/{title}' + '/{size}px--{title}.jpg`
with any size.
*Autoplay*: since this is a video element I guess we could directly start it after loading with .play()
`document.getElementById("test").play()`

**YouTube**
```
  const posterUrl = `https://i.ytimg.com/vi/${videoId}/${poster}.jpg`
  const posterUrlFallback = `https://i.ytimg.com/vi/${videoId}/hqdefault.jpg`
```
they also have an api that we could use if we register
*Autoplay*: autoplay=1 should work.

**Vimeo**
```https://vimeo.com/api/oembed.json?url=https://vimeo.com/432353``` gives url and size and more nice things. 👍 
*Autoplay*: `https://player.vimeo.com/video/${vimeo[3]}?autoplay=1` should work.

  
</details>

### PR DESCRIPTION
**Pull request by elbotho**
closes #631

### CODE REVIEW

**Comment by inyono** [comment]
Thanks, looks quite good already 👍 Going to continue the discussion in the issue.

**Comment by inyono** [comment]
IMHO: this is fine to merge after adding appropriate (i18n-)strings (i.e. the children of `InfoBar`)

File: src/components/content/privacy-wrapper.tsx
Comment (inyono) [inline src/components/content/privacy-wrapper.tsx]: @elbotho: Could you please add the "right" language here? Not sure how to do that since I didn't find a hook for that or something.

File: src/components/content/privacy-wrapper.tsx
Comment (inyono) [inline src/components/content/privacy-wrapper.tsx:39]: I removed the stuff with `alreadyConsented`. If we want that, we also need a way for users to withdraw their consent (e.g. by showing a similar message below the video).

File: src/components/content/privacy-wrapper.tsx
Comment (inyono) [inline src/components/content/privacy-wrapper.tsx]: IMHO: we really should review the i18n stuff soon. Even this rather simple case was super annoying to add and might not even work with all languages (e.g. different grammar).

File: src/components/content/privacy-wrapper.tsx
Comment (elbotho) [reply src/components/content/privacy-wrapper.tsx:39]: right, forgot about that. thanks.

File: src/components/content/privacy-wrapper.tsx
Comment (elbotho) [reply src/components/content/privacy-wrapper.tsx]: Simplified it a bit. Please check if it still works for you.
We still need to work on the implementation of course :)

**Review by inyono — APPROVED** [review]

**Pull request merged**
